### PR TITLE
Updated pic to Base64, Added Nu Validator Link 

### PR DESCRIPTION
**Pull request by AlphaAndTheOshea**
I updated my pic to base 64, I also added a link to Nu Validation of web page.

### CODE REVIEW

**Comment by AlphaAndTheOshea** [comment]
HOLLA AT YA BOI.

**Comment by AlphaAndTheOshea** [comment]
i should have all objectives completed for Dev-01, and Dev02 now. Please let me know if anything is wrong. @barrycumbie

**Comment by barrycumbie** [comment]
@AlphaAndTheOshea looks like you are getting the hang of git & github with this nice PR title and using the @ mentions. 

I've got a couple minor things...see my review and then I can approve. So extremely close!

why not give me a link to your active test page here in the comments too?

File: docs/dev-profile/harris-first.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <ol>
+          <li>GitHub</li>
+          <li>GitBash</li>
+          <li>Git command: ls - list all files ad folders in current directory</li>
+          <li>Git command: mkdir - makes a directory</li>
+          <li>Markdown language</li>
+        </ol>
+      </li>
+    </ul>
+    <a href="https://validator.w3.org/nu/?doc=https%3A%2F%2Falphaandtheoshea.github.io%2Fdev-challenge-01%2Fdocs%2Fdev-profile%2Fharris-first.html" target="_blank">Nu Html Checker</a>
```
Comment (barrycumbie) [inline docs/dev-profile/harris-first.html:36]: this needs to point to what your page WILL be when it is merged, so on barrycumbie.github.io (you are setting up a link now for a page that will exist in the future...very trippy)

**Review by barrycumbie — CHANGES_REQUESTED** [review]
can you remove the index.html from this PR...or initiate another one that just has ONE file?

**Comment by barrycumbie** [comment]
@AlphaAndTheOshea okay, okay, I see that you combined Dev 01 and Dev 02 into a single PR. Would prefer these to be separate as my earlier comment...

Dev 01 = a PR with just your dev provile
Dov 02 = PR with just than link added in the index.

**Comment by barrycumbie** [comment]
here is my test page Nu checked: https://validator.w3.org/nu/?doc=https%3A%2F%2Falphaandtheoshea.github.io%2Fdev-challenge-01%2Fdocs%2Fdev-profile%2Fharris-first.html

**Pull request closed without merging**